dascore/transform/strain.py

```diff
@@ -16,27 +19,85 @@ def velocity_to_strain_rate(
     order: int = 2,
 ) -> PatchType:
     """
-    Convert velocity das data to strain rate.
+    Convert velocity DAS data to strain rate.
 
     This is primarily used with Terra15 data and simply uses
     [patch.differentiate](`dascore.transform.differentiate.differentiate`)
-    under the hood.
+    under the hood which allows higher order differenciating.
+    It doesn't change the shape of the array.
 
     Parameters
     ----------
     patch
-        A patch object containing das data. Note: attrs['data_type'] should be
+        A patch object containing DAS data. Note: attrs['data_type'] should be
         velocity.
     gauge_multiple
         The multiples of spatial sampling to make the simulated gauge length.
+        Must be equal to 1 for now.
     order
         The order for the finite difference 1st derivative stencil (accuracy).
+
+    See Also
+    --------
+    See also [velocity_to_strain_rate_fd]\
+    (`dascore.transform.strain.velocity_to_strain_rate_fd`)
     """
     assert gauge_multiple == 1, "only supporting 1 for now."
     coord = patch.get_coord("distance", require_evenly_sampled=True)
-    step = coord.step
+    distance_step = coord.step
     patch = differentiate.func(patch, dim="distance", order=order)
+    data = patch.data / distance_step
```

Comment:
@d-chambers can you please double check if we need to divide by gauge length here?

Reply:
We don't actually need this; numpy.grad takes a dx parameter to properly normalize in that function call. This makes me feel better, I was worried the outputs were scaled wrong 😓.